### DISCUSSION TITLE
Simulate smoke ingress in cabin from engine fire in passengers aircrafts.

### DISCUSSION THREAD

**Discussion by sude-ep** [firemodels/fds#13149] [General]
How to model this ?? i tried hvac and exhauts methods but cannot get any result

**Comment by drjfloyd** [comment]
This isn't much detail to go on. Both approaches could be valid depending on your specific needs. I recommend you start simple. Just a small simple domain where you try various approaches and once you figure out and approach that works, then start adding complexity.

**Reply by sude-ep** [reply]
hey can i share my file ??

**Reply by mcgratta** [reply]
Do not share your file unless it is a relatively short and simple case that demonstrates a problem.
